packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -178,19 +178,25 @@ class NetworkController extends DisposableController
   }
 
   Future<void> loadOfflineData(OfflineNetworkData offlineData) async {
+    List<HttpProfileRequest> httpProfileData = [];
+    List<SocketStatistic> socketStatsData = [];
+
+    httpProfileData = offlineData.httpRequestData.mapToHttpProfileRequests;
+    socketStatsData = offlineData.socketData.mapToSocketStatistics;
```

Comment:
this can be condensed:
```
final httpProfileData = offlineData.httpRequestData.mapToHttpProfileRequests;
final socketStatsData = offlineData.socketData.mapToSocketStatistics;
```

Reply:
done